diff --git a/build.sbt b/build.sbt
index ad77b2d..4e05a96 100644
--- a/build.sbt
+++ b/build.sbt
@@ -171,3 +171,9 @@ val io = (project in file("io"))
     Test / buildInfoPackage := "sbt.internal.io",
     Test / buildInfoUsePackageAsPath := true,
   )
+
+commands ++= CommunityBuildPlugin.commands
+
+Global / resolvers += "The Scala Nightly Repository".at("https://repo.scala-lang.org/artifactory/maven-nightlies/")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
diff --git a/project/Dependencies.scala b/project/Dependencies.scala
index 359dbdb..5f77a86 100644
--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -4,7 +4,7 @@ import Keys._
 object Dependencies {
   val scala212 = "2.12.20"
   val scala213 = "2.13.16"
-  val scala3 = "3.3.6"
+  val scala3 = "3.10.0-RC1-bin-20260626-20f6657-NIGHTLY"
 
   val scalaCompiler = Def.setting {
     val v = if (scalaBinaryVersion.value == "3") scala213 else scalaVersion.value
diff --git a/project/plugins.sbt b/project/plugins.sbt
index bb675d1..5e19df5 100644
--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -5,3 +5,5 @@ addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.5.5")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.13.1")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.10.0")
 addSbtPlugin("org.scala-sbt" % "sbt-contraband" % "0.7.0")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
