diff --git a/build.sbt b/build.sbt
index 8613a25..c61a63a 100644
--- a/build.sbt
+++ b/build.sbt
@@ -1,4 +1,4 @@
-lazy val scala3                 = "3.2.2"
+lazy val scala3 = "3.8.4"
 lazy val scala213               = "2.13.10"
 lazy val supportedScalaVersions = List(scala3, scala213)
 lazy val scmUrl                 = "https://github.com/sky-uk/kafka-topic-loader"
@@ -65,3 +65,9 @@ addCommandAlias("checkFmt", "scalafmtCheckAll; scalafmtSbtCheck")
 addCommandAlias("runFmt", "scalafmtAll; scalafmtSbt")
 
 addCommandAlias("ciBuild", "checkFmt; checkFix; +test")
+
+commands ++= CommunityBuildPlugin.commands
+
+Global / resolvers += "The Scala Nightly Repository".at("https://repo.scala-lang.org/artifactory/maven-nightlies/")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
diff --git a/project/build.properties b/project/build.properties
index 46e43a9..e480c67 100644
--- a/project/build.properties
+++ b/project/build.properties
@@ -1 +1 @@
-sbt.version=1.8.2
+sbt.version=1.11.5
diff --git a/project/plugins.sbt b/project/plugins.sbt
index 158fb42..20fc7be 100644
--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -2,3 +2,7 @@ addSbtPlugin("org.scalameta"             % "sbt-scalafmt"   % "2.5.0")
 addSbtPlugin("ch.epfl.scala"             % "sbt-scalafix"   % "0.10.4")
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat"   % "0.4.1")
 addSbtPlugin("com.github.sbt"            % "sbt-ci-release" % "1.5.11")
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.14.2")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
