diff --git a/build.sbt b/build.sbt
index 700b180..aa677a4 100644
--- a/build.sbt
+++ b/build.sbt
@@ -56,3 +56,9 @@ lazy val sandbox = module
   .settings(libraryDependencies ++= Dependencies.sandbox)
   .settings(publish / skip := true)
   .dependsOn(prometheus4cats, `prometheus4cats-java`)
+
+commands ++= CommunityBuildPlugin.commands
+
+Global / resolvers += "The Scala Nightly Repository".at("https://repo.scala-lang.org/artifactory/maven-nightlies/")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
diff --git a/project/plugins.sbt b/project/plugins.sbt
index 07d71f0..f907dcf 100644
--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -14,3 +14,7 @@ addSbtPlugin("ch.epfl.scala"                     % "sbt-version-policy"    % "3.
 addSbtPlugin("org.typelevel"                     % "sbt-tpolecat"          % "0.5.2")
 addSbtPlugin("org.scalameta"                     % "sbt-mdoc"              % "2.8.2")
 addSbtPlugin("org.scalameta"                     % "sbt-scalafmt"          % "2.5.6")
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.14.2")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
