diff --git a/build.sbt b/build.sbt
index 52b4b10..8ce44be 100644
--- a/build.sbt
+++ b/build.sbt
@@ -1,4 +1,4 @@
-val scala3Version = "3.0.0"
+val scala3Version = "3.8.4"
 
 lazy val root = project
   .in(file("."))
@@ -14,3 +14,9 @@ lazy val root = project
       "com.novocode" % "junit-interface" % "0.11" % Test
     )
   )
+
+commands ++= CommunityBuildPlugin.commands
+
+Global / resolvers += "The Scala Nightly Repository".at("https://repo.scala-lang.org/artifactory/maven-nightlies/")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
diff --git a/project/build.properties b/project/build.properties
index 9edb75b..e480c67 100644
--- a/project/build.properties
+++ b/project/build.properties
@@ -1 +1 @@
-sbt.version=1.5.4
+sbt.version=1.11.5
diff --git a/project/plugins.sbt b/project/plugins.sbt
index d5a9f57..e51bab6 100644
--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -2,3 +2,5 @@ addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat"             % "0.1.20"
 addSbtPlugin("com.jsuereth"              % "sbt-pgp"                  % "2.0.1")
 addSbtPlugin("org.scalameta"             % "sbt-scalafmt"             % "2.4.0")
 addSbtPlugin("org.xerial.sbt"            % "sbt-sonatype"             % "3.9.7")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
