diff --git a/build.sbt b/build.sbt
index fba57b2..723f86d 100644
--- a/build.sbt
+++ b/build.sbt
@@ -835,3 +835,9 @@ lazy val nativeSettings: SettingsDefinition = List(
   Test / fork := false,
   coverageEnabled := false,
 )
+
+commands ++= CommunityBuildPlugin.commands
+
+Global / resolvers += "The Scala Nightly Repository".at("https://repo.scala-lang.org/artifactory/maven-nightlies/")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
diff --git a/project/plugins.sbt b/project/plugins.sbt
index 8f6d325..7095845 100644
--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -22,3 +22,7 @@ addSbtPlugin("io.kevinlee" % "sbt-devoops-scala"     % sbtDevOopsVersion)
 addSbtPlugin("io.kevinlee" % "sbt-devoops-sbt-extra" % sbtDevOopsVersion)
 addSbtPlugin("io.kevinlee" % "sbt-devoops-github"    % sbtDevOopsVersion)
 addSbtPlugin("io.kevinlee" % "sbt-devoops-starter"   % sbtDevOopsVersion)
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.14.2")
+
+ThisBuild / evictionErrorLevel := sbt.util.Level.Warn
